packages/sanity/src/core/store/bundles/types.ts

```diff
@@ -10,17 +9,23 @@ import {type bundlesReducerAction, type bundlesReducerState} from './reducer'
 /**
  * @internal
  */
-export interface BundleDocument extends SanityDocument {
+export interface BundleDocument {
```

Comment:
Instead of duplicating the fields like `_id` from `SanityDocument`, it might be better to `Pick` them.
```suggestion
export interface BundleDocument extends Pick<SanityDocument, '_id' | '_createdAt' | '_updatedAt' | '_rev' | '_version'> {
```

Alternatively, we could install type-fest or something similar to get [a helper that removes the index type for us](https://github.com/sindresorhus/type-fest/blob/main/source/omit-index-signature.d.ts):

```ts
import type {OmitIndexSignature} from 'type-fest'

export interface BundleDocument extends OmitIndexSignature<SanityDocument> {
```

(This is totally non-blocking, just a thought 🙂).